config/exchanges.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
@abdullahtariq91 i think its better if we add this file to git ignore? You can merge the PR after removing this one.